src/views/IDUploader/IDUploader.module.scss

```diff
@@ -41,11 +41,6 @@
       grid-row: 3;
```

Comment:
Did you by chance look at any of the other CSS in this file? If i'm remembershing correctly, a lot of it doesn't actually have an affect. See ../../SpreadsheetUploader/SpreadsheetUploader.Module.scss

Reply:
I didn't because that was not the goal of this change. I removed this class because I removed the only place it was used in it's component. Refactoring the ID Uploader is a separate project.